### PR TITLE
fix: model list display is broken

### PR DESCRIPTION
**Pull request by TheLLspectre**
Fix models window and it's scrollview.
Fix navigation inside pages, allow to get back the first pages, avoid to have multiple time the same pages in pagination system.
Display number of pages available.

- closes: #141

### CODE REVIEW

**Comment by TheLLspectre** [comment]
I try to check this we don't have the same result

image

**Comment by qvaleroo** [comment]
Maybe my Mac is the problem?

**Comment by TheLLspectre** [comment]
May be... I check it by the way, but we need @Morgan-6Freedom review to know

**Comment by TheLLspectre** [comment]
Done !

image

File: package/Editor/Models/ModelsUI.cs [hunk 1]
```diff
@@ -11,38 +11,51 @@ public class ModelsUI
...
 
+            currentPage++;
+
             var models = Models.GetModels();
         
-            if (firstImageIndex > models.Count - maxModelsPerPage)
+            if (firstImageIndex + maxModelsPerPage > models.Count)
             {
-                firstImageIndex = models.Count - maxModelsPerPage;
+                //firstImageIndex = models.Count - maxModelsPerPage;
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/ModelsUI.cs]: why comment this and not removing it ?

File: package/Editor/Models/ModelsUI.cs [hunk 2]
```diff
@@ -71,7 +95,8 @@ private void UpdatePage()
             
                 pageModels.Add(models[i]);
             }
-
+            numberPages = models.Count / maxModelsPerPage;
+            Debug.LogWarning("firstIndexImage: " + firstImageIndex + " " + models.Count + " " + maxModelsPerPage + " Count / maxModels = " + models.Count/maxModelsPerPage + " pageModels " + pageModels.Count);
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/ModelsUI.cs]: maybe you should remove this warning if it was only for easing debugging

**Review by Morgan-6Freedom — CHANGES_REQUESTED** [review]
Great.
If you fix what I suggested, it will be ok

File: package/Editor/Models/ModelsUI.cs (hunk 1 above)
Comment (TheLLspectre) [reply package/Editor/Models/ModelsUI.cs]: hmm they are no longer here

File: package/Editor/Models/ModelsUI.cs (hunk 2 above)
Comment (TheLLspectre) [reply package/Editor/Models/ModelsUI.cs]: And same for this one

**Comment by TheLLspectre** [comment]
I remove transparency @qvaleroo check if you already have the trouble on tags

File: package/Editor/Models/ModelsUI.cs (hunk 2 above)
Comment (Morgan-6Freedom) [reply package/Editor/Models/ModelsUI.cs]: are you sure you pushed the removing of the Debug.LogWarning ?

**Comment by qvaleroo** [comment]
Still transparent on my Mac: 
image

image

File: package/Editor/Models/ModelsUI.cs (hunk 2 above)
Comment (TheLLspectre) [reply package/Editor/Models/ModelsUI.cs]: Include in the commit from 2 days ago:
[fix: navigation in pages](https://github.com/scenario-labs/Scenario-Unity/pull/149/commits/62f61408115b73fab6ef9d7c7a40999f881fe183)

**Comment by qvaleroo** [comment]
image

Functionally OK 👌

**Review by Morgan-6Freedom — APPROVED** [review]

**Pull request merged**